app/shared/shared/services/table_mappings.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import List
+
+from shared.mapping.models import ScanReportTable
+from shared.services.storage_service import StorageService
+from shared_code import db
+from shared_code.models import VocabularyMapping
+
+logger = logging.getLogger("test_table_mapping_logger")
+
+storage_service = StorageService()
+
+
+def get_table_vocabulary_mappings(
```

Comment:
```suggestion
def get_field_vocabulary_mappings(
```
because we are trying to find which vocab ID goes with which SR field

Reply:
Sure. I'll fix it.